tree/tree/inc/TBasket.h

```diff
@@ -35,6 +34,8 @@ class TBranch;
 
 class TBasket : public TKey {
 
+   friend class TBasket_TestUnsupportedIO_Test;
```

Comment:
Is this used?

Reply:
Yes - unit test below.  `gtest` style is to include `gtest/gtest_prod.h` and do:

```
FRIEND_CLASS(TBasket, TestUnsupportedIO)
```

However, I didn't want to intermix `gtest` headers and ROOT public APIs...

Reply:
> However, I didn't want to intermix gtest headers and ROOT public APIs...

Yes, that would be worse :) ...